.github/workflows/main.yaml

```diff
@@ -5,58 +5,15 @@ on:
     branches: "*"
   pull_request:
     branches: master
-
+ schedule:
```

Comment:
The indentation appears to have broken the CI :(

Reply:
fixed f1fb775